tests/libnet/ipaddress.go

```diff
@@ -5,14 +5,14 @@ import (
 	netutils "k8s.io/utils/net"
 )
 
-func GetIp(ips []string, ipv6 bool) string {
+func GetIp(ips []string, family v1.IPFamily) string {
```

Comment:
again this whole commit is a refactor of other new commit of this PR,
this was discussed and from what i totally remember, we said a PR shouldnt refactor itself please

Reply:
I remember a different decision. @EdDev ?

Reply:
The agreement was to be flexible per the reviewer requests, as it affects how one reviews the changes. But at the same time to consider that a rebase/squash when a rename is needed may be too hard and can merit an exception.
When I am the (main) reviewer, I prefer it nit and clean, no review cycle commits and such workarounds as I do not feel it contributes to my review.. most of the time it actually makes it harder.

Bottom line, if you can squash it in the first commit that introduces this code in the first place, great. If it causes a great amount of work to handle the conflicts, then say so.
Generally speaking, refactoring the code you just posted in the same PR is a bit odd, but exceptions always exists.